scrapers/DEU/DE-BY/index.js

```diff
@@ -0,0 +1,79 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as datetime from '../../../lib/datetime.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  country: 'DEU',
+  state: 'DE-BY', // ISO 3166 notation
+  url: 'https://raw.githubusercontent.com/jgehrcke/covid-19-germany-gae/master/data.csv',
+  type: 'json',
+  timeseries: true,
+  curators: [
+    {
+      name: 'Dr. Jan-Philip Gehrcke',
+      email: 'jgehrcke@googlemail.com'
+    }
+  ],
+  sources: [
+    {
+      name: 'Robert Koch-Institut, Bavarian health ministry',
+      description: 'Fresh data obtained from Bavarian health ministry by ZEIT ONLINE',
+      url: 'https://github.com/jgehrcke/covid-19-germany-gae'
+    }
+  ],
+  async scraper() {
+    const data = await fetch.csv(this.url);
+
+    function rowToResult(row) {
+      return {
+        cases: parseInt(row['DE-BY_cases'], 10),
+        deaths: parseInt(row['DE-BY_deaths'], 10),
+        coordinates: [11.497, 48.79],
+        population: 13 * 10 ** 6
+      };
+    }
+
+    // location string for log and error msgs.
+    const loc = `${this.country}, ${this.state}`;
```

Comment:
You can use `geography.getName(this)` -- it works on any object with the relevant properties (`state`, `country`, etc)